skbio/tree/_tree.py

```diff
@@ -2717,6 +2717,52 @@ def compare_tip_distances(self, other, sample=None, dist_f=distance_from_r,
         return dist_f(self_matrix, other_matrix)
 
     @experimental(as_of="0.4.0")
+    def bifurcate(self, insert_length=None, remove_singles=True):
+        r"""Reorders the tree into a bifurcating tree, so that
```

Comment:
The first line of a docstring must be a single sentence within 79 characters, otherwise it will render incorrectly (note the `r` prefix is unnecessary here)